iroha_client/tests/subscribe_to_changes.rs

```diff
@@ -104,24 +113,33 @@ mod tests {
         let mut iroha_client = Client::new(&ClientConfiguration::from_iroha_configuration(
             &configuration,
         ));
-        iroha_client
-            .submit(create_asset.into())
-            .await
-            .expect("Failed to prepare state.");
-        while let Some(change) = stream.next().await {
-            println!("Change received {:?}", change);
-            match change {
-                Occurrence::Created(entity)
-                | Occurrence::Updated(entity)
-                | Occurrence::Deleted(entity) => match entity {
-                    Entity::Transaction(_) => {
-                        println!("Entity changed: {:?}", entity);
-                        return ();
+        task::spawn(async move {
+            task::sleep(Duration::from_millis(300)).await;
+            iroha_client
+                .submit(create_asset.into())
+                .await
+                .expect("Failed to prepare state.");
+        });
+        async_std::future::timeout(
```

Comment:
For such cases you should import `use async_std::feature` and invoke it via `future::timeout`